src/interface_layer/src/__main__.py

```diff
@@ -0,0 +1,269 @@
+# Import external tools
+from logging.handlers import RotatingFileHandler
+from logging import Formatter, DEBUG
+from flask import Flask, request
+from json import loads
+from configparser import ConfigParser as CP
+import logging
+from json import dumps
+from requests import post, Response
+# Import local tools
+from NgacApi.ngac import NGAC
+from result import Result, Ok, Error, unwrap, is_error
+from NgacApi import *
+from NgacApi.access_request import AccessRequest

```

Comment:
Would it not be better to update the `__init__.py` in `NgacApi` to make fewer imports?
So in `NgacApi` > `__init__.py` it should be something like: 
```python
from .access_request import AccessRequest
from .policy import Policy
```
This would mean that we instead of having 10 different `from NgacApi` we can use one.
So the import would look something like this:
```python
from NgacApi import AccessRequest, Policy, ..., etc.
```

Reply:
That would make it cleaner, and will fix